setup.py

```diff
@@ -22,5 +24,6 @@
         'matplotlib',
         'sklearn',
         'keras-model-specs>=0.0.28',
+        'setuptools<=39.1.0'
```

Comment:
this is only needed to work with that tensorflow version, am I right?

Reply:
I think this is the setuptools version for both `1.10` and `1.11`. From my understanding, Setuptools is a package manager which should help in getting the correct versions of packages. 

If this line isn't there you get a mismatch of setuptools version and tensorflow version.